wvc-ourwork-nextjs/components/Charts/BarChart/BarChart.tsx

```diff
@@ -47,15 +47,41 @@ export function BarChart({
       },
       title: {
         display: title,
-        text: fullTitle,
-        position: 'bottom',
+        text: titlePosition === 'top' ? title : subTitle,
+        position: titlePosition,
+        color:
+          titlePosition === 'top'
+            ? !isDarkMode
+              ? ''
+              : '#ffffff'
+            : !isDarkMode
+            ? '#666666'
+            : '#cccccc',
```

Comment:
Im finding it a bit tough to follow what's going on here - is there any other way we could write this? No worries if there isn't a quick refactor, maybe even some comments would help

Reply:
Ah yeah, it took a loong time to figure this one out too when I was doing it! Since there are nested layers, title subtitle, dark mode or not. Added some comments ⭐️